src/ResourceManager/Compute/Commands.Compute.ManagedService/Tools/Move-AzureRmVhdVM.ps1

```diff
@@ -0,0 +1,237 @@
+<#
+.ExternalHelp AzureRM.Compute.ManagedService-help.xml
+#>
+
+function Move-AzureRmVhdVM
+{
```

Comment:
I don't think we have figured out how to take the necessary depencency, or whether this shoudl be a cmdlet or a script